apiserver/plane/bgtasks/notification_task.py

```
@@ -286,7 +293,7 @@ def notifications(
 
             # ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- #
             issue_subscribers = list(
-                IssueSubscriber.objects.filter(project_id=project_id, issue_id=issue_id)
+                IssueSubscriber.objects.filter(project_id=project_id, issue_id=issue_id, project__project_projectmember__is_active=True,)
```

Ruff

Line too long (137 > 88)

(E501)